dae/dae/gpf_instance/gpf_instance.py

```diff
@@ -160,7 +160,7 @@ def _pheno_db(self):
     def gene_scores_db(self):
```

Comment:
I tend to avoid annotating results in `GPFInstance`. The reason is to postpone the import of the corresponding modules (check `grr`, `gene_models`, `reference_genome`, `genotype_storages`, etc.).

Alternatively, if the `gpf_instance` loads all needed GPF packages at the beginning of the module, it most certainly will lead to circular imports.

In this particular case, it is possible to add type annotation, but I'd rather hide the `GeneScoreDb` import inside the `gene_scores_db` property definition and skip the type annotation.